sympy/physics/mechanics/kane.py

```diff
@@ -86,6 +86,9 @@ class KanesMethod(_Methods):
         The particles and rigid bodies in the system.
     forcelist : iterable of tuple[Point | ReferenceFrame, Vector], optional
         Forces and torques applied on the system.
+    reaction_forces: iterable containing the reaction forces, optional.
+        Needed only to eliminate them in dynamic part of (fr + frstar)
+        and the force vector
```

Comment:
Why is it necessary to pass in new information? Wouldn't KanesMethod already have all the necessary information inside it?

Reply:
> Why is it necessary to pass in new information? Wouldn't KanesMethod already have all the necessary information inside it?

I would not know, how to find them. My thinking was also this: If KanesMethod always knew about them, why would they sometimes be put where they should not be?

Reply:
I think that all forces are given to KanesMethod, so they should be available inside the class. You'll need to figure out where they are and then make you code work with that. I doubt there is any need to introduce a new kwarg.